apps/api/src/migration/1676577091787-RemoveOldUnusedTables.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class RemoveOldUnusedTables1676577091787 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      DROP TABLE IF EXISTS form;
+      DROP TABLE IF EXISTS submission;
+      DROP TABLE IF EXISTS applicant_status;
+      DROP TABLE IF EXISTS applicants;
+      DROP TABLE IF EXISTS applicant_status_audit;
+      DROP TABLE IF EXISTS applicant_audit;
```

Comment:
I am a bit concerned about the applicant/applicant_status/applicant_audit tables having relationships between them, preventing a delete. Would you be able to add "CASCADE" here?

Reply:
hmm, wonder if it's better I just take these 4 out?  I don't know about these tables relations, do you recall if they were referencing any existing/ in use tables?

Reply:
nevermind, found the commit the migrations were removed, ref [here](https://github.com/bcgov/internationally-educated-nurses/pull/33/files#:~:text=apps/api/src/migration/1645638796025%2DApplicantAndStatus.ts), looks like they only reference each other